content/1.7/back-office/orders/delivery-slips/view-delivery-slips.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Add / edit attribute value
+weight: 1
+---
+
+# Delivery slips
+
+In this page 2 sections are present:
+
+## Print PDF
+
+2 Input fields are presented:
+
+1) From
+
+Input field for the date, by default has the current day entered. Has a tooltip: "Format: 2011-12-31 (inclusive)". When pressed opens a calendar to choose a date.
+
+2) To
+
+Input field for the date, by default has the current day entered. Has a tooltip: "Format: 2011-12-31 (inclusive)". When pressed opens a calendar to choose a date.
+
+### Call to action
+
+ - Generate PDF, when pressed, if no PDF was done on the selected time, will show the message: "No delivery slip was found for this period."
```

Comment:
And when the PDF is generated ? 
Where do we explain how is generate the PDF ?

Reply:
@fabienvallon should we describe the ones in the back office or the ones that the user can check in his FO account?